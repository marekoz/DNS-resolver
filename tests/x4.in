140.82.121.3 -x  -r -s kazi.fit.vutbr.cz