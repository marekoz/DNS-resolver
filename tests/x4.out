Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
  3.121.82.140.in-addr.arpa., Type: PTR, Class: IN
Answer section (1)
  3.121.82.140.in-addr.arpa., Type: PTR, Class: IN, TTL: <number>, Data length: 32, lb-140-82-121-3-fra.github.com.
Authority section (0)
Additional section (0)